environment.yml

```diff
@@ -35,3 +35,5 @@ dependencies:
   - cartopy
   - doc8
   - recommonmark
+  - pip:
```

Comment:
Let's vendor in pooch temporarily while things settle just a bit and we work on things like conda packages. Without conda packages for pooch, we won't even be able to update MetPy's packages.

Reply:
Yeah, I thought about that. I think I'll get started on the conda-forge package and we make new releases later with fixes required by metpy.